### ISSUE TITLE
Error installing v0.14.0-unstable0143

### ISSUE THREAD

**Issue by pascalberger** [chocolatey/ChocolateyGUI#350]
Running `choco install chocolateygui --source https://www.myget.org/F/packages_develop/api/v2 --pre`, which gets me `v0.14.0-unstable0143`, fails with the following error message:

 ```
WARNING: Write-ChocolateyFailure is deprecated and will be removed in v2. If you are the package maintainer, please use 'throw $_.Exception' instead.
ERROR: Running ["C:\Windows\System32\msiexec.exe" /i "C:\ProgramData\chocolatey\lib\ChocolateyGUI\tools\ChocolateyGUI.msi" /quiet] was not successful. Exit code was '1603'. See log for possible error messages.
The install of chocolateygui was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\ChocolateyGUI\tools\chocolateyInstall.ps1'.
 See log for details.
```

The MSI log in the Chocolatey temp folder shows:

```
Chocolatey must already be installed.
```

Also running the MSI directly fails with a popup mentioning the above error.

Chocolatey Version is: Chocolatey v0.10.3 Professional

**Comment by gep13** [comment]
@pascalberger do you have Chocolatey installed in a custom location?  Or is it installed to default?

**Comment by pascalberger** [comment]
Default location (installed using the command provided on the Chocolatey website)

**Comment by RichiCoder1** [comment]
Apologies. This was due to a flawed checked. Fixing now.

**Comment by RichiCoder1** [comment]
Fixed: ec8fe869c5ce722ba6dbfe0f981f5b4e6b35b617
